### PR TITLE
add footer and fix css

### PR DESCRIPTION
**Pull request by bkm412**

### CODE REVIEW

File: src/components/local/exchange/BalanceA.js [hunk 1]
```diff
@@ -27,16 +28,16 @@ export default class BalanceA extends React.Component {
...
                     <tbody>
                     <tr className="first-row">
-                        <td className="symbol">ETH</td>
+                        <td className="symbol">{this.props.baseName}</td>
                         <td className="able">
                             <p>*ETH available</p>
                         </td>
                         <td className="total">
-                            <p>*ETH total</p>
+                            <p>{}</p>
```
Comment (aman-mahalgavan) [inline src/components/local/exchange/BalanceA.js]: what are the empty curly braces for?

File: src/components/local/exchange/BalanceA.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ export default class BalanceA extends React.Component {
     }
 
     render() {
+        console.log(this.props)
```
Comment (aman-mahalgavan) [inline src/components/local/exchange/BalanceA.js]: Remove all console.logs please

File: src/components/local/exchange/InstrumentSelectA.js [hunk 3]
```diff
@@ -26,21 +27,95 @@ export default class InstrumentSelectA extends React.Component {
...
-            this.setState({ sortType: `up-${type}` });
+            this.setState({sortType: `up-${type}`});
         }
     }
 
+    doSort(type, coin) {
+        if (!coin) {
+            return [];
+        }
+        // const volume = this.state.volume;
```
Comment (aman-mahalgavan) [inline src/components/local/exchange/InstrumentSelectA.js]: If we do not need a reference to the old code then please remove the commented code

File: src/utilities/config.js [hunk 4]
```diff
@@ -3,75 +3,86 @@ export default {
...
             tokenAddress: "0x0000000000000000000000000000000000000000",
             decimal: 18,
             prTrade: [
                 {
                     productName: "BAT",
+                    symbolName: "BAT",
                     decimal: 18,
                     productId: 2,
                     tokenAddress: "0x4514fd2f873b859a5d713eba6dd9047a58a0da5e",
                     logo: ""
```
Comment (aman-mahalgavan) [inline src/utilities/config.js]: we can put the logo path here and use it for rendering the token logo on the UI

File: src/components/local/exchange/BalanceA.js (hunk 1 above)
Comment (bkm412) [reply src/components/local/exchange/BalanceA.js]: I changed it to bind data

File: src/components/local/exchange/BalanceA.js (hunk 2 above)
Comment (bkm412) [reply src/components/local/exchange/BalanceA.js]: I deleted it. Thank you

File: src/components/local/exchange/InstrumentSelectA.js (hunk 3 above)
Comment (bkm412) [reply src/components/local/exchange/InstrumentSelectA.js]: I deleted it. Thank you

**Pull request merged**